.github/workflows/masterfhirvalidation.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Check out IOPS-Validation
         uses: actions/checkout@master
         with:
-          repository: NHSDigital/IOPS-Validation
+          repository: NHSDigital/IOPS-FHIR-Test-Scripts # orignally known as NHSDigital/IOPS-Validation
```

Comment:
I dont think you need to check this out as this is the repo you're working in, it does check it out to a new folder `validation` and then `npm` builds stuff, but the runner is transient.  I can't see anything to suggest that this would cause an issue over using the one that exists.  The repo NHSDigital/IOPS-FHIR-Validation-Service is checked out and put in the folder validation-service-fhir-r4, so there doesn't seem to be anything that would be overwritten by not putting a new copy in a folder.

This would allow you to remove this step, and also remove all the -C switches from the calls to make.

Reply:
I tried removing this but from checking using ls command the folder does not appear unless the repo is checked out

Reply:
This bit checkouts the current repo, which is `NHSDigital/IOPS-FHIR-Test-Scripts`

![image](https://github.com/NHSDigital/IOPS-FHIR-Test-Scripts/assets/93662162/73163c47-20d0-4ec4-8fe5-e841b445ada8)

So you what you are doing with this

![image](https://github.com/NHSDigital/IOPS-FHIR-Test-Scripts/assets/93662162/a617cb0a-3fc0-45b8-aaa5-076fe9d4f392)

Is checking it out again and putting it in a folder called validation.

Good call on just removing the makefile, it was adding nothing.  So for example, in this step you cd into `validation`

![image](https://github.com/NHSDigital/IOPS-FHIR-Test-Scripts/assets/93662162/7786db7b-fb04-4311-af5f-726d5629e43b)

you could just call `npm ci`

Reply:
Also, the second one checks out master, so you are only ever running whats in the master branch, which isn't going to be helpful if work on this repo started using feature branches

Reply:
Actually I forgot this is pulled into other repos, so that initial checkout is probably checking out the repo you are pulling it into, so in that case you would still need to check it out, and putting it in another folder makes sense then.  Little be confusing, but reusing the same action script for this across different repos makes sense.  Going to resolve this and approve as I dont think there is anything else to clean up.